src/SFA.DAS.EmployerApprenticeshipsService.Application/Queries/GetTrainingProgrammes/GetTrainingProgrammesQueryHandler.cs

```diff
@@ -13,20 +13,20 @@ public sealed class GetTrainingProgrammesQueryHandler : IAsyncRequestHandler<Get
 
         public GetTrainingProgrammesQueryHandler(IApprenticeshipInfoServiceWrapper apprenticeshipInfoServiceWrapper)
         {
-            if (apprenticeshipInfoServiceWrapper == null)
-                throw new ArgumentNullException(nameof(apprenticeshipInfoServiceWrapper));
-
             _apprenticeshipInfoServiceWrapper = apprenticeshipInfoServiceWrapper;
         }
 
         public async Task<GetTrainingProgrammesQueryResponse> Handle(GetTrainingProgrammesQueryRequest message)
```

Comment:
We could avoid creating a dummy task, FrameworksView and List here, e.g.
`
IEnumerable<ITrainingProgramme> programmes;
var standardsTask = _apprenticeshipInfoServiceWrapper.GetStandardsAsync();
if (!message.IncludeFrameworks)
{
  programmes = (await standardsTask).Standards;
}
else
{
  var getFrameworksTask = _apprenticeshipInfoServiceWrapper.GetFrameworksAsync();
  programmes = (await standardsTask).Standards.Union((await getFrameworksTask).Frameworks.Cast<ITrainingProgramme>());
}

return new GetTrainingProgrammesQueryResponse
{
  TrainingProgrammes = programmes.OrderBy(m => m.Title).ToList()
};
`